packages/js/product-editor/src/components/attribute-control/new-attribute-modal.scss

```diff
@@ -34,6 +34,20 @@
 				position: absolute;
 				z-index: 10;
 			}
+
+			.components-form-token-field__token.is-validating {
+				animation: components-button__busy-animation 2s linear infinite;
+				background-image: linear-gradient(-45deg, #fafafa 33%, #e0e0e0 0, #e0e0e0 70%, #fafafa 0);
+				background-size: 100px 100%;
+				box-shadow: 0 0 1px black inset;
+				opacity: 1;
+
+				.components-form-token-field__token-text,
+				.components-form-token-field__remove-token {
+					background-color:transparent;
+					pointer-events: none;
+				}
+			}
```

Comment:
It would be nice to push this state and styling upstream to the core token field component in a future PR.

Reply:
I'm unsure if core wants to apply these styles. We can ask to confirm.